components/forms/button.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="section--bg-white" style="height: 250px; padding: 2rem;">
```

Comment:
If we want to simply our stuff further, we could take these inline styles here and below and create some classes just for the pattern library in the preview.css file.